test/exec/child.js

```diff
@@ -1,3 +1,5 @@
 // does nothing child process
 console.log("Child process.id: "+process.pid);
 console.log(" - - - - - - - - - - - - - - - - - - - - - - - ");
+// keep process alive for one minute
+setTimeout(function () {}, 60000);
```

Comment:
Maybe it's not necessary, I got confused because the windows tests are still very timing-dependent, but I was surprised the tests were ever working because the child processes will exit once they finish those two `console.log`s.  Using `setTimeout` keeps the process alive.  The timeout definitely doesn't have to be for one minute though.  It could just be a bit longer than the duration of the tests.